daemon/src/projection.rs

```diff
@@ -290,8 +293,9 @@ impl Cfd {
                 (CfdState::ContractSetup, vec![])
             }
             ContractSetupCompleted { dlc } => {
-                self.details.tx_url_list.push(TxUrl::new(
-                    dlc.lock.0.txid(),
+                self.details.tx_url_list.push(TxUrl::from_transaction(
+                    &dlc.lock.0,
+                    &dlc.script_pubkey_for(role),
```

Comment:
📣 The lock transaction has a single output which does not belong to either party and corresponds to a shared output, so this won't highlight anything.